src/main/java/io/jenkins/plugins/opentelemetry/semconv/JenkinsOtelSemanticAttributes.java

```diff
@@ -30,6 +30,7 @@
      * @see hudson.model.Node#getDisplayName() ()
      */
     public static final AttributeKey<String>        CI_PIPELINE_AGENT_NAME = AttributeKey.stringKey("ci.pipeline.agent.name");
+    public static final AttributeKey<List<String>>  CI_PIPELINE_RUN_CULPRITS = AttributeKey.stringArrayKey("ci.pipeline.run.culprits");
```

Comment:
Should we use a different term rather than culprits? Thinking about the standardisation. @mdelapenya pointed to use `committers` IIRC.

Though, we can change it in the future anyway, since we will probably revisit the attribute names

Reply:
if it's just listing the committers of the changes of this build, I like the idea to call the attribute `CI_PIPELINE_RUN_COMMITTERS`